README.md

```diff
@@ -37,9 +37,12 @@ $ docker run --log-driver=sumologic --log-opt sumo-url=https://<deployment>.sumo
 ### Sumo Logic Options
 To specify additional logging driver options, you can use the `--log-opt NAME=VALUE` flag.
 
-| Option                      | Required? | Default Value | Description
-| --------------------------- | :-------: | :-----------: | -------------------------------------- |
+| Option                    | Required? | Default Value | Description
+| ------------------------- | :-------: | :-----------: | -------------------------------------- |
 | sumo-url                  | Yes       |               | HTTP Source URL
+| sumo-source-category      | No        |               | Source category to appear when searching on Sumo Logic by `_sourceCategory`. Use `{{Tag}}` as the placeholder of `tag` option. If not specified, the source category of HTTP source will be used
+| sumo-source-name          | No        |               | Source name to appear when searching on Sumo Logic by `_sourceName`. Use `{{Tag}}` as the placeholder of `tag` option. If not specified, it will be the container's name
```

Comment:
"If not specified, it will be the container's name" This should be in the "Default Value" column it sounds like

Reply:
it's a little bit different, since default value is always a fixed value but these 3 options are actually a behavior (since you cannot specify a `container's name` no matter whatever string you assigned here)

Reply:
I woudl suggest using `code style` for the fixed value ones, and regular text description for others.

From casual user's perspective, if the "Default Value" column is empty, then they would expect no default value.

Reply:
ok. make sense